src/components/Documents/DocumentTableRow.jsx

```diff
@@ -27,7 +28,7 @@ import DOC_TYPES from '../../constants/doc_types';
  * @param {documentTableRowProps} Props - Props for DocumentTableRow
  * @returns {React.JSX.Element} The DocumentTableRow component
  */
-const DocumentTableRow = ({ document }) => {
+const DocumentTableRow = ({ document, handlePermissions }) => {
```

Comment:
Would need to update the typedef for `documentTableRowProps` to include `handlePermissions`